### PR TITLE
[TIMELINE] Removing extra fields in many-to-many between Themes and Measures

### PR DESCRIPTION
**Pull request by Remg**
- ManyToMany between Theme and Measure does not need extra fields.

- property `Measure.global` is renamed to `Measure.major` to be consistent with the UI.

- the Themes should be flagged in the Measure admin, not the other way.

- some Timeline admin fix/improvements

### CODE REVIEW

File: src/Admin/TimelineMeasureAdmin.php [hunk 1]
```diff
@@ -59,6 +62,27 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
...
+                'label' => 'Statut',
+                'show_filter' => true,
+                'field_type' => ChoiceType::class,
+                'field_options' => [
+                    'multiple' => true,
+                    'choices' => Measure::STATUSES,
+                ],
+            ])
+            ->add('major', null, [
+                'label' => 'Mise en avant (32)',
```
Comment (HeahDude) [inline src/Admin/TimelineMeasureAdmin.php]: Why `(32)`? Forgotten debug or specific request?

File: src/Entity/Timeline/Theme.php [hunk 2]
```diff
@@ -73,20 +74,29 @@ class Theme implements EntityMediaInterface
...
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\Timeline\Measure", mappedBy="themes")
+     *
+     * @Algolia\Attribute
      */
     private $measures;
 
-    public function __construct()
-    {
+    public function __construct(
+        ?string $title = null,
```
Comment (HeahDude) [inline src/Entity/Timeline/Theme.php]: The `?` is needless when the parameter is set to `null` by default.

File: src/Admin/TimelineMeasureAdmin.php (hunk 1 above)
Comment (Remg) [reply src/Admin/TimelineMeasureAdmin.php]: They just want `Mise en avant (32)` this to be the label of this specific field.

Comment (Remg) [reply src/Admin/TimelineMeasureAdmin.php]: nice catch though, after update they are consistent in every admin now

**Pull request merged**